test/Poseidon/GoldenTestsRunCommands.hs

```diff
@@ -313,13 +313,18 @@ testPipelineForge testDir checkFilePath = do
         , "ForgePac2" </> "ForgePac2.janno"
         ]
 
+ -- Note: We here use our test server (no SSL and different port). The reason is that 
+ -- sometimes we would like to implement new features that affect the communication
+ -- between server and client, and we need tests succeeding before Pull Requests are merged, so
+ -- we adopt the policy to run experimental builds on the test server in order to test features
+ -- before running them on the main server.
```

Comment:
So... the test server isn't something I'd publicly announce. It's more of an internal vehicle to test new server versions and to test new datasets: We have two repos on the server, one "staging", and one "stable"... if there is a new dataset commit or push, I'll pull first into staging and restart the test server to see whether I get any errors, and then if things work fine, pull into stable and restart the main server.

I wasn't gonna document this, because for users, the main server is the only thing that's needed.

That said, I'm aware that I need to somewhere document the server setup, so we can replicate it if we need to move elsewhere, and - in the long term - to enable others to take over maintenance in case of a personal emergency or so. 

That documentation needs to be somewhere in our Poseidon-framework team, but I think it shouldn't be public.

Reply:
I wasn't gonna use the test server officially, but now I realised it's needed in our test code, and hence it's visible in public code... I can personally live with the fact that this is sort of an internal URL that's not publicly documented...

Reply:
Sounds all sensible to me. 

An alternative test setup might be to start a server locally just for the test. That would make the software fully independent of any external server, which may or may not be available.

Reply:
Aaah. Indeed! Very nice idea! And not too hard to implement, actually... Let's think about that. I'll open a "nice to have" issue.